dev-util/bcc/bcc-0.22.0-r1.ebuild

```diff
@@ -25,8 +26,8 @@ RDEPEND="
 	>=dev-libs/elfutils-0.166:=
 	>=dev-libs/libbpf-0.5.0:=[static-libs(-)]
 	>=sys-kernel/linux-headers-5.13
-	<=sys-devel/clang-14:=
-	<=sys-devel/llvm-14:=[llvm_targets_BPF(+)]
+	<=sys-devel/clang-$((${LLVM_MAX_SLOT} + 1)):=
```

Comment:
Note that we need <, not <=, if we're doing ```... + 1```.

The reason being that for say, LLVM_MAX_SLOT=13, we want to allow 13.0.1, and so on, but not 14.

Reply:
yes, overlooked that `<=` was still present